tests/placement-rotation.test.html

```diff
@@ -0,0 +1,38 @@
+<html>
+  <body>
+    <div data-ea-publisher="test"></div>
+
+    <script type="module">
+      import { expect } from "@open-wc/testing";
+      import { runTests } from "@web/test-runner-mocha";
+
+      import { wait, detectedKeywords } from "../index";
+      import { mockAdDecision } from "./common.inc";
+
+      let stub = mockAdDecision();
+
+      runTests(async () => {
+        describe("EthicalAds library", () => {
+          it("placement rotation", async () => {
```

Comment:
This is basically just testing the rotation "works", and not any of the logic except the hash change under the time limit, right?  Definitely a great start :100:

Reply:
Correct. We'd have to change the time limit somehow in testing to test the hash change.